pipelines/respondent-home-ui.yml

```diff
@@ -345,6 +395,15 @@ jobs:
     file: ras-deploy/tasks/extract-source-from-release.yml
     input_mapping: { release-resource: respondent-home-ui-release }
     output_mapping: { release-source: respondent-home-ui-release-source }
+  - put: create-redis
+    resource: cf-cli-resource-prod
+    params:
+      command: create-service
+      service: awselasticache-redis
+      plan: small
+      service_instance: ras-redis
```

Comment:
Just did a quick check and it looks like the existing `service_instance` in prod is called `res-ops-redis`. This will mean we probably wouldn't need to create a new instance as well now?